engo_glfw.go

```diff
@@ -33,13 +33,43 @@ var (
 	canvasWidth, canvasHeight float32
 )
 
+// WindowResizeMessage is a message that's being dispatched whenever the game window is being resized by the gamer
+type WindowResizeMessage struct {
```

Comment:
glfw isn't the only platform we support so this being in this file breaks all the other platforms.